loadtest/store/store.go

```diff
@@ -11,6 +11,8 @@ type UserStore interface {
 	Id() string
 	// TODO: Move all getters to this interface
 
+	// Config return the server configuration settings
+	Config() *model.Config
```

Comment:
Ideally this should return a copy cause as it is it's mutable but given it's a config object which is only going to be used from the outside I think it's fine to return like this, at least for now.

Reply:
Changed. Let me know it it's ok :D

Reply:
Sorry, what I meant is that even like this it's still mutable cause `model.Config` is a deeply nested struct full of pointers. We would have to make a deep copy to make it really immutable but it's okay like this for now. We can merge this :+1: